.github/workflows/docker.yml

```diff
@@ -0,0 +1,19 @@
+name: Build and push docker images
+
+env:
+  OWNER: ${{ github.repository_owner }}
+
+on: [push]
+jobs:
+  amd64-base:
+    uses: ./.github/workflows/docker-build-push.yml
+    with:
+      platform: amd64
+      resulting_tag: 'latest'
+      jupyter_tag: 'latest'
```

Comment:
The `resulting_tag` is what is pushed to Dockerhub as `osmnx:resulting_tag` while the `jupyter_tag` is what is pulled in the Dockerfile's `FROM jupyter/base-notebook:$JUPYTER_TAG`

Could use the same argument for both to enforce strict compliance with the Jupyter notebook team naming convention, but wanted to float this option.